File: day6/javascript/agecalculator.js
function calulateAge(dateOfBirthStr) {
	const formatter = 'DD-MM-YYYY';
	const dateOfBirth = moment(dateOfBirthStr, formatter);
	const today = moment();
	const age = today.diff(dateOfBirth, 'years');
	return age;

	}

const dateOfBirth = '20-04-1991"
const age = calculateAge(dateOfBirth);
console.log('Age: ', age);

File: day6/java/AgeCalculator.java
import java.time.LocalDate;
import java.time.Period;
import java.time.format.DateTimeFormatter;
import java.util.Scanner;

public class AgeCalculator {

	public static int calculateAge(String dateOfBirthStr) {

		DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd-MM-yyyy");

		LocalDate dateOfBirth = LocalDate.parse(dateOfBirthStr , formatter);

		LocalDate today = LocalDate.now();

		Period period = Period.between(dateOfBirth, today);

		return period.getYears();

	} 


}